src/components/Radio/RadioItem/RadioItem.module.css

```diff
@@ -0,0 +1,102 @@
+.radioInput {
+  height: 0;
+  width: 0;
+  opacity: 0;
+  position: absolute;
+}
+
+.radioInput + .radioButtonContainer {
```

Comment:
Why did you add `.radioInput +` in front?

Reply:
Because a lot of the selectors contain .radioInput:disabled or radioInput:checked. To me this made it more readable to keep the same basic structure.